src/tactic/ui/tools/ingest_wdg.py

```diff
@@ -1013,15 +1050,27 @@ def execute(my):
         input_prefix = update_data.get('input_prefix')
         
         for count, filename in enumerate(filenames):
+            
+            # Check if files should be updated. 
+            # If so, attempt to find one to update.
+            # If more than one is found, do not update.
+            if update_mode in ["true", "True"]:
+                # first see if this sobjects still exists
+                search = Search(search_type)
+                search.add_filter("name", filename)
+                if relative_dir and search.column_exists("relative_dir"):
+                    search.add_filter("relative_dir", relative_dir)
```

Comment:
I think we should add this so it's scoped by this current project
search.add_project_filter()